### DISCUSSION TITLE
liaison entre routeurs

### DISCUSSION THREAD

**Discussion by ghost** [dbarzin/mercator#242] [Q&A]
Bonjour,

Je cherche à faire une cartographie de mon SI avec votre outil qui est vraiment intéressant. Cependant, j'ai remarqué (ou alors raté l'information) qu'on ne peut pas lier un routeur logique avec un routeur physique ou lier plusieurs routeurs physiques ou/et logiques entre eux. Dans mon SI, je possède plusieurs routeurs qui ont un lien direct entre eux et je ne trouve pas la possibilité de les liés. Ai-je raté une étape ou cela n'est juste pas possible ? De plus j'aimerai faire de même avec les switchs (commutateurs) physiques et ou logiques

Cordialement,

RODRIGUEZ Erwan

**Comment by dbarzin** [comment]
Bonsoir,
Mercator ne permet pas de représenter le plan de routage de l'infrastructure réseau. 
La cartographie du système d'information ne doit-elle pas s'arrêter à certaines limites techniques ?
Ces informations sont-elles pertinentes pour la cartographie ?
Faut-il tout mettre dans la cartographie ? Y compris les règles de routage ?
Faudra-t-il mettre également les règles de filtrage firewall ?
Une intégration des fonctions offertes par [Mininet Editor](https://github.com/Thomaash/me) est envisagée.
Qu'en penses-tu ?
Cordialement,
Didier

**Reply by ghost** [reply]
Bonjour,

Je me suis mal exprimé. Je ne cherche pas quelque chose d'aussi poussé. Je ne cherche pas à représenter un plan de routage ou à mettre les règles de filtrages firewall. Je cherche juste à faire une cartographie me permettant de faire une étude d'impact.
Est-ce qu'on pourrait rajouter un lien entre le router physique et logique ? Et faire de même pour les commutateurs logiques et physiques ? Car par exemple, si un routeur brûle, l'étude d'impact qu'offre la cartographie me permettra de voir rapidement l'impact que cela aurait sur le reste de l'infrastructure à la fois coté physique et logique par la vue exploration. 
Je vous remercie pour la découverte de l'outil Miniter Editor. Je vais prendre le temps de l'étudier.

Cordialement,

RODRIGUEZ Erwan

**Reply by dbarzin** [reply]
Merci Erwan,
Cela va être ajouté à la feuille de route.
Didier

**Comment by dbarzin** [comment]
Cette amélioration a été implémentée !
Cordialement,
Didier
